recipes/ignition-cmake/all/conanfile.py

```diff
@@ -30,15 +34,21 @@ def _configure_cmake(self):
         self._cmake = CMake(self)
         self._cmake.definitions["CMAKE_INSTALL_DATAROOTDIR"] = "lib"
         self._cmake.definitions["SKIP_component_name"] = False
-
         self._cmake.configure(source_folder=self._source_subfolder)
         return self._cmake
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        os.rename(glob.glob("ign-cmake*")[0], self._source_subfolder)
+        version_major = self.version.split(".")[0]
+        conan.tools.files.rename(
+             self, f"gz-cmake-ignition-cmake{version_major}_{self.version}",
```

Comment:
```suggestion
             self, f"gz-cmake-ignition-cmake{version_major}_{self.version}",
```
Is not destination parameter enough? https://docs.conan.io/en/latest/reference/tools.html#tools-get

Reply:
hi @uilianries  I do not understand your suggestion clearly. Can you precisely tell what am I doing wrong here ?

Reply:
Use `tools.get`only, dont need os.rename.